### ISSUE TITLE
[Feature] Upload studio favicon

### ISSUE THREAD

**Issue by echo6ix** [stashapp/stash#2082]
**Describe the solution you'd like**
Add ability for users to upload a 16x16 favicon to each studio on the studio (edit) page.

I think about two years ago there was talk about adding studio website favicons to Trizkat's repo. Either way, scrapping favicons is easy, and a lot of the major studios use one. Future iterations of Stash could use the favicon in more appropriate areas of the UI where a large logo looks out of place.

Example concept where you could use a favicon...

Video scene page (favicon displayed at the top left adjacent to scene title)
![1](https://user-images.githubusercontent.com/37937507/143987808-08710c4f-a0e3-4148-9716-ec37e3d2e7ff.png)

Video scene page (all info fields expanded)
![2](https://user-images.githubusercontent.com/37937507/143987843-99ef6c62-cc6e-467c-a170-5838ec1df56d.png)

Studio edit page (showing favicon upload form)
![8XV3n8N6BcDMPD1bkymV7M](https://user-images.githubusercontent.com/37937507/143988152-3648809a-df15-4880-b6f9-0ea58c2b188b.png)

**Comment by feederbox826** [comment]
related: #5399

**Comment by echo6ix** [comment]
> related: [#5399](https://github.com/stashapp/stash/issues/5399)

I'm not sure this is meaningfully related. FR #5399 is asking for additional favicons to display on performer URL links. This FR is asking for the capacity to have an additional studio image that specifically supports the favicon format.

**Comment by feederbox826** [comment]
If 5399 were to be implemented, I suspect that a dynamic table for favicon/url mappings would fare much better than manually adding every one manually, and this would follow the sample principle of studio url -> favicon mappings.

Pro-Am Studios like OnlyFans or Fansly would have the performer/studio favicons duplicated otherwise, so a shared or generic mapping is the only one that really makes sense

**Comment by echo6ix** [comment]
@feederbox826 Fair enough. Now that I've searched, this FR #5094 is a much more comprehensive duplicate. I suspect it could be the primary FR that all related FRs point to.
